crates/figma_import/src/reaction_schema.rs

```diff
@@ -265,19 +289,44 @@ impl From<TriggerJson> for Trigger {
     }
 }
 
+#[derive(Deserialize, Serialize, Debug, Copy, Clone, PartialEq)]
+pub enum Easing {
+    Bezier(Bezier),
+    Spring(Spring),
+}
+
 // We flatten the Easing type to a bezier for the toolkit. These values were taken from
 // https://easings.net/ and verified against Figma optically.
-impl From<EasingJson> for Bezier {
+impl From<EasingJson> for Easing {
     fn from(json: EasingJson) -> Self {
         match json {
-            EasingJson::EaseIn => Self { x1: 0.12, y1: 0.0, x2: 0.39, y2: 0.0 },
-            EasingJson::EaseOut => Self { x1: 0.61, y1: 1.0, x2: 0.88, y2: 1.0 },
-            EasingJson::EaseInAndOut => Self { x1: 0.37, y1: 0.0, x2: 0.63, y2: 1.0 },
-            EasingJson::Linear => Self { x1: 0.0, y1: 0.0, x2: 1.0, y2: 1.0 },
-            EasingJson::EaseInBack => Self { x1: 0.36, y1: 0.0, x2: 0.66, y2: -0.56 },
-            EasingJson::EaseOutBack => Self { x1: 0.34, y1: 1.56, x2: 0.64, y2: 1.0 },
-            EasingJson::EaseInAndOutBack => Self { x1: 0.68, y1: -0.6, x2: 0.32, y2: 1.6 },
-            EasingJson::CustomCubicBezier { bezier } => bezier,
+            EasingJson::EaseIn => Self::Bezier(Bezier { x1: 0.12, y1: 0.0, x2: 0.39, y2: 0.0 }),
+            EasingJson::EaseOut => Self::Bezier(Bezier { x1: 0.61, y1: 1.0, x2: 0.88, y2: 1.0 }),
```

Comment:
I guess those curves have control points that are only at the bottom or top. I'd much rather the Figma API just returns the values instead of these symbolic names -- now we have to keep them in sync if Figma changes the definitions.

Reply:
Oh, I didn't know you found these values from Figma. Yea that's a bit annoying. How did you find the values then?

Reply:
We can always ask them to enhance their API

Reply:
Yeah -- let's add it to our Figma API wish list doc :). 

For these values, I took the cubic bezier control points from a very old reference (someone called Robert Penner named these values, Macromedia Flash developers popularized them, now they're in Figma, unchanged), and eyeballed that we seemed to show the same output as Figma's preview viewer.

For the spring values, I took them from Figma's blog post announcing the spring timing function.

I'll make a ticket to expand the comment to explain how to keep these in sync.

Reply:
#976